### ISSUE TITLE
RuntimeException: Unable to destroy activity NullPointerException: Attempt to invoke virtual method 'void android.view.View.dispatchDetachedFromWindow()' on a null object reference

### ISSUE THREAD

**Issue by mikemike396** [barteksc/AndroidPdfViewer#66]
Updated to 2.0 and getting this error when hitting the back button from the PDF activity. Everything else loads up fine and the PDF view works great just when I try to exit that activity it crashes. I didn't change anything else besides remove the scroll bar. Thanks for the help and awesome library!

```
   pdfView.fromFile(newFile)
                .defaultPage(0)
                .scrollHandle(new DefaultScrollHandle(getActivity()))
                .enableAnnotationRendering(true)
                .load();
```

```
W/OpenGLRenderer: Failed to set EGL_SWAP_BEHAVIOR on surface 0xdf214680, error=EGL_SUCCESS
E/Surface: getSlotFromBufferLocked: unknown buffer: 0xdf92dc80
D/jniPdfium: Destroy FPDF library
D/AndroidRuntime: Shutting down VM
E/AndroidRuntime: FATAL EXCEPTION: main
                                                                            
java.lang.RuntimeException: Unable to destroy activity {myapp.PDFViewer.PDFView_activity}: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.view.View.dispatchDetachedFromWindow()' on a null object reference
                                                                                 at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:3831)
                                                                                 at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:3849)
                                                                                 at android.app.ActivityThread.-wrap5(ActivityThread.java)
                                                                                 at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1398)
                                                                                 at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                                 at android.os.Looper.loop(Looper.java:148)
                                                                                 at android.app.ActivityThread.main(ActivityThread.java:5417)
                                                                                 at java.lang.reflect.Method.invoke(Native Method)
                                                                                 at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
                                                                                 at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
                                                                              Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.view.View.dispatchDetachedFromWindow()' on a null object reference
                                                                                 at android.view.ViewGroup.dispatchDetachedFromWindow(ViewGroup.java:3063)
                                                                                 at android.view.ViewGroup.removeViewInternal(ViewGroup.java:4603)
                                                                                 at android.view.ViewGroup.removeViewInternal(ViewGroup.java:4576)
                                                                                 at android.view.ViewGroup.removeView(ViewGroup.java:4507)
                                                                                 at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1165)
                                                                                 at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1252)
                                                                                 at android.support.v4.app.FragmentManagerImpl.moveToState(FragmentManager.java:1234)
                                                                                 at android.support.v4.app.FragmentManagerImpl.dispatchDestroy(FragmentManager.java:2083)
                                                                                 at android.support.v4.app.FragmentController.dispatchDestroy(FragmentController.java:244)
                                                                                 at android.support.v4.app.FragmentActivity.onDestroy(FragmentActivity.java:368)
                                                                                 at android.support.v7.app.AppCompatActivity.onDestroy(AppCompatActivity.java:203)
                                                                                 at android.app.Activity.performDestroy(Activity.java:6407)
                                                                                 at android.app.Instrumentation.callActivityOnDestroy(Instrumentation.java:1142)
                                                                                 at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:3818)
                                                                                 at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:3849) 
                                                                                 at android.app.ActivityThread.-wrap5(ActivityThread.java) 
                                                                                 at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1398) 
                                                                                 at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                                 at android.os.Looper.loop(Looper.java:148) 
                                                                                 at android.app.ActivityThread.main(ActivityThread.java:5417) 
                                                                                 at java.lang.reflect.Method.invoke(Native Method) 
                                                                                 at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
                                                                                 at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
I/Process: Sending signal. PID: 9432 SIG: 9
```

**Comment by barteksc** [comment]
Fixed in version 2.0.1

**Comment by mikemike396** [comment]
Looks good thank you!
